### PR TITLE
CCITCARBON 233

### PR DESCRIPTION
**Pull request by rujutashinde**
CCITCARBON 233 Update teflo config code to not make defaults section in teflo.cfg file manadatory
* modified config.py
* updated core.py to add read_file

### CODE REVIEW

File: teflo/core.py [hunk 1]
```diff
@@ -1516,7 +1513,7 @@ def create_master(self, all_hosts):
             # load it and keep building upon it
             if os.path.exists(self.master_inv):
                 with open(self.master_inv) as f:
-                    config.readfp(f)
+                    config.read_file(f)
```
Comment (JunqiZhang0) [inline teflo/core.py:1516]: how is it related to the change "not make defaults section in teflo.cfg file manadatory"? is it required for this change?

File: teflo/utils/config.py [hunk 2]
```diff
@@ -21,18 +21,13 @@
     Teflos own config module for loading configuration settings defined by
     the user.
 
-    :copyright: (c) 2017 Red Hat, Inc.
+    :copyright: (c) 2020 Red Hat, Inc.
```
Comment (JunqiZhang0) [inline teflo/utils/config.py:24]: it's better in a separate commit to fix these kind of typo as we need clean commits

File: teflo/utils/pipeline.py [hunk 3]
```diff
@@ -22,7 +22,7 @@
     Module containing classes and functions for building pipelines of resources
     and tasks for teflo to run.
 
-    :copyright: (c) 2017 Red Hat, Inc.
+    :copyright: (c) 2020 Red Hat, Inc.
```
Comment (JunqiZhang0) [inline teflo/utils/pipeline.py:25]: could be in a separate commit and pr

**Review by JunqiZhang0 — CHANGES_REQUESTED** [review]
Looks good to me except for several changes that seem to be "not required"

**Comment by rujutashinde** [comment]
Closing this PR and sending separate PRs as per the comments.
